cmd/get/getCluster.go

```diff
@@ -37,96 +37,90 @@ import (
 	"github.com/liggitt/tabwriter"
 )
 
+// TODO : deal with --all flag to manage differentiate started cluster and stopped cluster like `docker ps` and `docker ps -a`
```

Comment:
Out of scope for this PR, but still worth mentioning: during development of k3d v1.x, we had this differentiation and the community didn't like it, so we made `--all` the default.
I think we should rather stick to `kubectl`-similar style and show all clusters, when no flag is set and only filter them by setting new flags.

Reply:
Yes i think i will update getkubeconfig in another PR to remove `--all` flag. I am not sure to understand, k3d style should show only started cluster without `--all` or not ?

Reply:
I made proposal for this change here https://github.com/rancher/k3d/pull/268.